index.js

```diff
@@ -0,0 +1,8 @@
+const http = require('http')
+
+http.createServer(function (request, response) {
+  console.log('request url', request.url)
+  console.log('request ', JSON.stringify(request.url))
+}).listen(8125)
```

Comment:
Port is defined in the config

Reply:
Removed the file: https://github.com/commercetools/commercetools-adyen-integration/pull/28/commits/5e83220996de4bd095edff0d76bdd5efa5eabfe3